samples/client/petstore/java/microprofile-rest-client-3.0/src/main/java/org/openapitools/client/api/PetApi.java

```diff
@@ -52,7 +52,7 @@ public interface PetApi  {
     
     @Consumes({ "application/json", "application/xml" })
     @Produces({ "application/xml", "application/json" })
-    public Pet addPet(Pet pet) throws ApiException, ProcessingException;
+    Pet addPet(Pet pet) throws ApiException, ProcessingException;
```

Comment:
why `public` is removed? shall we keep it?

Reply:
Because it is default for interfaces and can (should) be omitted.

Reply:
👌